### DISCUSSION TITLE
Does a GH release require new/more permissions than before?

### DISCUSSION THREAD

**Discussion by helpermethod** [jreleaser/jreleaser#1393] [Q&A] [answered]
For some unknown reason, I'm getting a 403 when releasing to GitHub, staging my artifacts to Maven Central seems to work, though:

https://github.com/helpermethod/zip-forge/actions/runs/5061455689/jobs/9085699708#step:4:183

I remember that the `secrets.GITHUB_TOKEN` was enough to release to GH, but I also remember that some projects required additional permissions at some point.

**Comment by aalmiray** [comment]
I wonder if it's related to https://github.com/jreleaser/jreleaser/issues/1377

**Reply by helpermethod** [reply]
I'm not even pushing a tag, I'm just using Maven's CI-friendly version. Is the `JRELEASER_PROJECT_VERSION` required?

**Reply by aalmiray** [reply, marked as answer]
`JRELEASER_PROJECT_VERSION` is not needed. You can see at line 123 of the `Build` step that project version was set to `0.1.0`.
Here's another probable cause -> https://github.com/jreleaser/jreleaser.github.io/issues/54

The repository might not have enough permissions.

**Reply by helpermethod** [reply]
That was the problem! Assigning read/write permissions and everything works instantly! Thanks alot!
